IDE/STM32Cube/wolfssl_example.c

```diff
@@ -105,7 +105,8 @@
 #endif /* WOLFSSL_STATIC_MEMORY */
 
 
-/* UART definitions */
+/* This sets which UART to use for the console.  It is something you will have
+ * to configure in STMCubeIDE and then change here. */
```

Comment:
Can you indicate this comes from the `wolfSSL.I-CUBE-wolfSSL_conf.h` files, which is generated based on the Cube configuration and specifically the `default_conf.ftl` in the pack.